src/System.Runtime.WindowsRuntime.UI.Xaml/tests/System.Runtime.WindowsRuntime.UI.Xaml.Tests.csproj

```diff
@@ -24,5 +24,11 @@
       <Link>Common\System\AssertExtensions.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\src\System.Runtime.WindowsRuntime.UI.Xaml.csproj">
```

Comment:
@danmosemsft this seems like an odd thing to have to do to get the test project compiling